client/src/components/DevInfo.js

```diff
@@ -38,12 +38,12 @@ export default function DevInfo({ authorInitial, authorName, authorBio, authorIm
         </Typography>
       </CardContent>
       <CardActions disableSpacing>
-        <IconButton aria-label="github">
-          <GitHubIcon onClick={() => window.open(gitHub, "_blank")} />
+        <IconButton aria-label="github" onClick={() => window.open(gitHub, "_blank")}>
```

Comment:
could you explain why you changed that?

Reply:
From what I understand, <IconButton> is the "parent" to the other <...Icon> such as <GitHubIcon>, <LinkedInIcon>, <ShareIcon> which are the child. Based on this, the original code had the onClick effect on the CHILD instead of the PARENT which is what is causing the error to appear. The original error reads: 
"You are providing an onClick event listener to a child of a button element. Prefer applying it to the IconButton directly. This guarantees that the whole <button> will be responsive to click events."
With this in mind (as well as searching online at stackoverflow) the appropriate solution would be to simply move the onClick events to the parent instead of leaving them in the child.